CanalTP/Chaos
# Add sort impact search order by application end date

# Description

This PR add the sort parameter "impactApplicationEndDate" 
See [Swagger](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/87279b91808b830270a2b6aa0b8c78b4ca93c6bc/documentation/swagger.yml#/Search/post_impacts__search)

## Issue

Issue link: BOT-2990

## How to test

Here are the following steps to test this pull request:

- create some impacts
- use the search impact API with the new parameter (see swagger)
- check when change the order type that the response is in correct sorting state
